include/fw_metadata.h

```diff
@@ -199,6 +201,36 @@ fw_firmware_info_get(u32_t firmware_address)
 	return NULL;
 }
 
+#define FW_INFO_OFFSET0 0x200
```

Comment:
Vector tables are always the same size on the same hardware. From what I've seen, we can't go much smaller, but we could look more into it.

Reply:
On nrf91, the vector table seems to take 0x40 in total, but the total space used in front of the fw_info is 0x144 in a lot of cases (maybe IRQ handlers etc?) I think 0x200 is reasonable, but it could be squeezed a bit. What do you think?